pkg/component/observability/monitoring/alertmanager/alertmanager_test.go

```diff
@@ -236,15 +235,23 @@ var _ = Describe("Alertmanager", func() {
 								corev1.ResourceMemory: resource.MustParse("20Mi"),
 							},
 							MaxAllowed: corev1.ResourceList{
-								corev1.ResourceCPU:    resource.MustParse("500m"),
 								corev1.ResourceMemory: resource.MustParse("200Mi"),
 							},
-							ControlledValues: &vpaControlledValuesRequestsOnly,
+							ControlledValues:    &vpaControlledValuesRequestsOnly,
+							ControlledResources: &[]corev1.ResourceName{corev1.ResourceMemory},
 						},
 						{
 							ContainerName: "config-reloader",
 							Mode:          &vpaContainerScalingModeOff,
 						},
+						{
+							ContainerName: "oauth2-proxy",
+							Mode:          &vpaContainerScalingModeOff,
+						},
+						{
+							ContainerName: "kube-rbac-proxy",
+							Mode:          &vpaContainerScalingModeOff,
+						},
```

Comment:
AFAIK, these sidecars are injected by the [oidc-apps-controller](https://github.com/gardener/oidc-apps-controller). Can we move the mutation there?

Reply:
> AFAIK, these sidecars are injected by the [oidc-apps-controller](https://github.com/gardener/oidc-apps-controller). Can we move the mutation there?

Yes, it is possible. The oidc-apps-controller injects configuration for the corresponding sidecars in https://github.com/gardener/oidc-apps-controller/blob/0bda0a0627de66bba80cff56fa061b5b80ae059f/pkg/webhook/vpa-webhook.go#L77-L106.